.github/workflows/prettier.yml

```
@@ -1,6 +1,11 @@
 name: Prettier
 
-on: [pull_request]
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
```

```suggestion
on: [pull_request]
```
Change this back, because you don't need to run prettier just because someone edited the PR's title.

---

aw just wanted to be sure about perms but sure